README.md

```diff
@@ -352,6 +352,25 @@ The default port the Pushgateway is listening to is 9091. The path looks like:
 
         curl -X PUT http://pushgateway.example.org:9091/api/v1/admin/wipe
         
+## Management API
+
+Pushgateway provides a set of management API to ease automation and integrations.
+
+* Available endpoints:
+ 
+| HTTP_METHOD |  PATH | DESCRIPTION |
+| :-------: | :-----| :----- |
+| GET    | /-/healthy |  Return 200 whenever the Pushgateway is healthy. |
+| GET    | /-/ready |  Return 200 whenever the Pushgateway is ready. |
```

Comment:
Perhaps "… is ready to serve traffic."?

Reply:
I would use 3rd person, i.e. "Returns" instead of "Return" for both lines above.